docs/reference/google.auth.crypt.rst

```diff
@@ -1,7 +1,16 @@
-google.auth.crypt module
-========================
+google.auth.crypt package
+=========================
```

Comment:
Good spot!

Reply:
No credit to me :) the auto doc functions caught this.